shared/src/business/entities/EntityConstants.ts

```diff
@@ -600,28 +654,6 @@ export const STATUS_REPORT_ORDER_DOCUMENTS_ALLOWLIST = uniq(
     .map(x => x.eventCode),
 );
 
-export const EXTERNAL_TRACKED_DOCUMENT_EVENT_CODES =
-  EXTERNAL_DOCUMENTS_ARRAY.filter(
-    doc =>
-      doc.category === TRACKED_DOCUMENT_TYPES.application.category ||
-      doc.category === TRACKED_DOCUMENT_TYPES.motion.category,
-  ).map(x => x.eventCode);
-export const INTERNAL_TRACKED_DOCUMENT_EVENT_CODES =
-  INTERNAL_DOCUMENTS_ARRAY.filter(
-    doc =>
-      doc.category === TRACKED_DOCUMENT_TYPES.application.category ||
-      doc.category === TRACKED_DOCUMENT_TYPES.motion.category,
-  ).map(x => x.eventCode);
-
-export const TRACKED_DOCUMENT_TYPES_EVENT_CODES = union(
-  EXTERNAL_TRACKED_DOCUMENT_EVENT_CODES,
-  INTERNAL_TRACKED_DOCUMENT_EVENT_CODES,
-  [
-    TRACKED_DOCUMENT_TYPES.proposedStipulatedDecision.eventCode,
-    TRACKED_DOCUMENT_TYPES.orderToShowCause.eventCode,
-  ],
-);
```

Comment:
I moved this up to be next to the other logic for `TRACKED_DOCUMENT_TYPES`. I also refactored slightly and removed unused exports.